vstools/utils/clips.py

```diff
@@ -138,12 +141,14 @@ def initialize_clip(
     :param color_range:     ColorRange prop to set. If None, tries to get the ColorRange from existing props.
                             If no props are set, assume Limited Range.
     :param field_based:     FieldBased prop to set. If None, tries to get the FieldBased from existing props.
+                            If no props are set, assume PROGRESSIVE.
```

Comment:
if you say the default for this, you might as well for everything else and it'd be verbose
remove